site/content/en/docs/drivers/includes/vmware_windows_usage.inc

```diff
@@ -0,0 +1,13 @@
+## Requirements
+
+*VMware Workstation
```

Comment:
There should be a space after the asterisk, to make this a bullet point. See: https://deploy-preview-9937--kubernetes-sigs-minikube.netlify.app/docs/drivers/vmware/